test/new_relic/agent/instrumentation/rails/active_support_subscriber.rb

```diff
@@ -163,42 +159,6 @@ def test_pop_segment_returns_false
           end
         end
 
-        def test_start_logs_notification_error
```

Comment:
Moved these tests into the notification_subscriber_test since these are testing what is now shared code